consensus/src/chained_bft/consensus_types/block.rs

```diff
@@ -263,6 +284,10 @@ where
     pub fn is_genesis_block(&self) -> bool {
         self.id() == *GENESIS_BLOCK_ID
     }
+
+    pub fn is_nil_block(&self) -> bool {
+        self.author == ValidatorSigner::<Ed25519PrivateKey>::nil_signing_key().author()
```

Comment:
We will need to identify the source of a NIL block at all times, if we foresee accountability for proposing a useless block. 

Since we have a block synchronization strategy, a validator may (and should) have the possibility of being a messenger, but not the originator of a block => we can't rely on network attributability.

Could we have a multi-sig here, with #n signers == 2?
Better, yet, since the NIL block is just a bit saying ignore content, could we have a convention around signing a nil-transaction?